go.mod

```diff
@@ -2,6 +2,8 @@ module antrea.io/ofnet
 
 go 1.15
 
+replace antrea.io/libOpenflow => /home/avarma/work/git-ashish-varma/libOpenflow
```

Comment:
No need to replace, the patch in libOpenflow is already merged.

Reply:
oops. This slipped in the commit by accident. Would remove it.